test/helpers/request.js

```diff
@@ -1,8 +1,8 @@
 const request = require('supertest');
 const nock = require('nock');
-const ServiceUrlGetter = require('../../src/services/service-url-getter');
+const forestUrlGetter = require('../../src/utils/forest-url-getter');
```

Comment:
[THINKING] You might want to use the `inject()` methods here, knowing that the file might be deleted